packages/at_secondary_server/lib/src/connection/outbound/outbound_message_listener.dart

```diff
@@ -50,7 +50,9 @@ class OutboundMessageListener {
       }
     } else {
       _buffer.clear();
-      throw BufferOverFlowException('Buffer overflow on outbound connection');
+      throw BufferOverFlowException('OutboundBuffer overflow: server received'
```

Comment:
server sent.. not received in outbound message listener

Reply:
message has been updated in the latest commit